env/tasks/main.yml

```diff
@@ -3,5 +3,5 @@
 - name: Ensure that a SQL database is available
   assert:
     that:
-      - 'gitlab__database in [ "postgresql", "mariadb" ]'
+      - 'gitlab__database == "postgresql" ]'
```

Comment:
You need to remove the leftover `]` character after the list was changed to a string.